test/phoenix_live_view/html_engine_test.exs

```diff
@@ -593,6 +595,24 @@ defmodule Phoenix.LiveView.HTMLEngineTest do
       end)
     end
 
+    test "when tag is unclosed" do
+      message = """
+      test/phoenix_live_view/html_engine_test.exs:2:1: end of template reached without closing tag for <div>
+        |
+      1 | <div>Foo</div>
+      2 | <div>
+        |    ^\
```

Comment:
The code in this PR looks great but I think we found a bug in the tokenizer/engine because this should point to the last character (i.e. `>`).

Reply:
I guess the mistake was in the code_snippet function. We were calculating the arrow padding using the padding from eex since I just copy/pasted it from there. Should be fixed now!